pkg/kubelet/cm/topologymanager/numa_info.go

```diff
@@ -0,0 +1,151 @@
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package topologymanager
+
+import (
+	"fmt"
+	"io/ioutil"
+	"strconv"
+	"strings"
+
+	cadvisorapi "github.com/google/cadvisor/info/v1"
+	"k8s.io/kubernetes/pkg/kubelet/cm/topologymanager/bitmask"
+)
+
+const (
+	defaultNodeDir = "/sys/devices/system/node/"
+)
+
+type NUMADistances [][]uint64
+
+type NUMAInfo struct {
+	Nodes         []int
+	NUMADistances NUMADistances
+}
+
+type NUMASysFs struct {
+	nodeDir string
+}
+
+func NewNUMAInfo(topology []cadvisorapi.Node) (*NUMAInfo, error) {
+	return newNUMAInfo(topology, &NUMASysFs{nodeDir: defaultNodeDir})
+}
+
+func newNUMAInfo(topology []cadvisorapi.Node, sysFs *NUMASysFs) (*NUMAInfo, error) {
```

Comment:
If you are no longer using a "fake" `sysFs` in the tests, then having this helper function is no longer necessary. Simply create `sysFs` as a local variable inside `NewNUMAInfo()` just before using it.

Reply:
Ack. I just did a quick search for `newNUMAInfo` in my browser and didn't see any other references because github had collapsed the view on `numa_info_test.go` because of its size. I see it now. Thanks.